### ISSUE TITLE
LAN Access Post Install

### ISSUE THREAD

**Issue by j-schwartz** [rachelproject/rachelplus#22]
I haven't been able to access my CAP over LAN post running first three scripts.  It does have a LAN IP address, however I get a time_out error message when navigating to the device from a different client

ifconfig output:

eth0      Link encap:Ethernet  HWaddr f8:35:dd:f3:d2:81  
          inet addr:192.168.1.109  Bcast:192.168.1.255  Mask:255.255.255.0
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:52031 errors:0 dropped:0 overruns:0 frame:0
          TX packets:28 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:3491042 (3.4 MB)  TX bytes:1776 (1.7 KB)

lo        Link encap:Local Loopback  
          inet addr:127.0.0.1  Mask:255.0.0.0
          UP LOOPBACK RUNNING  MTU:65536  Metric:1
          RX packets:77545 errors:0 dropped:0 overruns:0 frame:0
          TX packets:77545 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:0 
          RX bytes:4522903 (4.5 MB)  TX bytes:4522903 (4.5 MB)

wlan0     Link encap:Ethernet  HWaddr f8:35:dd:eb:3c:c4  
          inet addr:192.168.88.1  Bcast:192.168.88.255  Mask:255.255.255.0
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metric:1
          RX packets:3719755 errors:109447 dropped:0 overruns:0 frame:0
          TX packets:36307 errors:2 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:1000 
          RX bytes:931916061 (931.9 MB)  TX bytes:9321139 (9.3 MB)
          Interrupt:16 Base address:0x8000 

Google Chrome could not load the webpage because 192.168.1.109 took too long to respond. The website may be down, or you may be experiencing issues with your Internet connection.
Check your Internet connection
Check any cables and reboot any routers, modems, or other network devices you may be using.
Allow Chrome to access the network in your firewall or antivirus settings.
If it is already listed as a program allowed to access the network, try removing it from the list and adding it again.
If you use a proxy server...
Check your proxy settings or contact your network administrator to make sure the proxy server is working. If you don't believe you should be using a proxy server: Go to the Chrome menu > Settings > Show advanced settings... > Change proxy settings... > LAN Settings and deselect "Use a proxy server for your LAN".

**Comment by bnordha** [comment]
I don't have any issue at all connecting via the WAN port. It is actually
my default connection method.
Are you able to ping and ssh to it using that IP address?

Bernd

**Comment by spatiald** [comment]
After running the first install script via the line below (which will automatically run all of the correct scripts), the core functionality hasn’t changed.  I tested with a couple clean installs and I can still get to the default Intel CAP site on port 80 (via Google Chrome) and via SSH.  The only other thing you might have an issue with is the Google cache. Try clearing your history?

**Comment by j-schwartz** [comment]
Thanks. I don't know why but on a third install it worked fine. It may have
been a cache issue. I'm not sure.

I think the core works pretty great except for captive portal. I get a 404
error when I do get "caught" by the captive portal. (50/50 the portal grabs
me and directs me to a new browser window. The address it sends me to is
my.content, which doesn't redirect when entered into a browser either.
Leads me to believe it's a my.content issue.

**Comment by spatiald** [comment]
There is a timeout on the captive portal before it will redirect you again.  It’s built into prevent annoying you if you reconnect from a wireless moment break.  That’s probably the captive portal issue you are experiencing.  I believe the timeout is 30 seconds.

Using only the initial install, you should get a 404 as we having uploaded our content shell yet nor the captive portal content.  The post install should fix that and take you to our splash page.

My intent is to add the post install script to start after the core scripts once it’s functional.  Very close on that.

**Comment by j-schwartz** [comment]
Awesome.  Very cool.  Great work on this stuff, it's very fun.

**Comment by spatiald** [comment]
Try the script from scratch…I believe it should build the complete skeleton RACHEL with captive portal splash.

It defaults to UK repos but I am working that fix and still working the USB recovery issues we have.

Let me know

**Comment by hackersforcharity** [comment]
Let me clarify…use the first install script…the post install was merged with script 3.

**Comment by bnordha** [comment]
I uncommented the lines for the repos, and then selected the SG repo. It worked well for me.

 

Total installation took less than 5min.

**Comment by spatiald** [comment]
I updated the core install script along with script 1 and 3.  Many updates:
- Cleaned up code to clone/update contentshell files from GitHub
- Added fix for iptables to enable redirect from 10.10.10.10 to 192.168.88.1
- Install line changed to the one below (fixed on main page); just copy it into the CAP console and it will install
- Install does NOT overwrite /media/RACHEL/rachel content (if there); it WILL update the contentshell files to those that are on GitHub

Run with:
```bash
wget https://github.com/rachelproject/rachelplus/raw/master/cap-rachel-first-install.sh \
-O /root/cap-rachel-first-install.sh; bash cap-rachel-first-install.sh
```

**Comment by spatiald** [comment]
Reopen, if needed.  This issue is resolved by running the core script here:
```bash
wget https://github.com/rachelproject/rachelplus/raw/master/cap-rachel-first-install.sh \
-O /root/cap-rachel-first-install.sh; bash cap-rachel-first-install.sh
```
